src/vm-memory/src/mmap.rs

```diff
@@ -380,6 +478,21 @@ impl GuestMemoryMmap {
 
         Err(Error::InvalidGuestRegion)
     }
+
+    /// Mutably apply the specified function on each memory region, including the dirty bitmap.
+    pub fn with_regions_bitmaps_mut<F>(&self, mut cb: F) -> result::Result<(), GuestRegionMmapError>
```

Comment:
Is this needed ? Can't we just call `with_regions_mut` and call `region.bitmap()` for each region ?

Reply:
You are right, we can do that.